src/Parser/Mage/Arcane/Modules/Features/Abilities.js

```diff
@@ -0,0 +1,236 @@
+import SPELLS from 'common/SPELLS';
+
+import CoreAbilities from 'Parser/Core/Modules/Abilities';
+
+class Abilities extends CoreAbilities {
+  spellbook() {
+    const combatant = this.combatants.selected;
+    return [
+      // Rotational spells
+      {
+        spell: SPELLS.ARCANE_BLAST,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      },
+      {
+        spell: SPELLS.ARCANE_MISSILES,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      },
+      {
+        spell: SPELLS.ARCANE_BARRAGE,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      },
+      {
+        spell: SPELLS.ARCANE_EXPLOSION,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      },
+      {
+        spell: SPELLS.MARK_OF_ALUNETH,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 60,
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.90,
+        },
+      },
+      {
+        spell: SPELLS.ARCANE_FAMILIAR_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 10,
+        enabled: combatant.hasTalent(SPELLS.ARCANE_FAMILIAR_TALENT.id),
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.90,
+        },
+      },
+      {
+        spell: SPELLS.SUPERNOVA_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 25,
+        enabled: combatant.hasTalent(SPELLS.SUPERNOVA_TALENT.id),
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.90,
+        },
+      },
+      {
+        spell: SPELLS.CHARGED_UP_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 40,
+        enabled: combatant.hasTalent(SPELLS.CHARGED_UP_TALENT.id),
+        castEfficiency: {
+          suggestion: true,
+          recommendedEfficiency: 0.90,
+        },
+      },
+      {
+        spell: SPELLS.CHARGED_UP_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+        cooldown: haste => 12 / (1 + haste),
+        enabled: combatant.hasTalent(SPELLS.LIVING_BOMB_TALENT.id),
+      },
```

Comment:
Should this be living bomb? You have charged up talent twice here

Reply:
woops, missed that. should be fixed now